tests/test_hyperbolic.py

```diff
@@ -70,7 +70,7 @@ def test_projection_to_tangent_space(self):
         result = self.metric.inner_product(tangent_vec, base_point)
         expected = 0.
 
-        self.assertAllClose(result, expected)
+        self.assertAllClose(result, expected, atol=1e-2)
```

Comment:
why this `atol`? wasn't it working before?

Reply:
Unfortunately yes.

Reply:
It was a mistake. Fixed !

Reply:
It was a mistake. Fixed !